lib/helpers.js

```diff
@@ -123,6 +125,17 @@ export const parseResults = (editor, results, filePath, showIgnored) => {
   return toReturn;
 };
 
+export const saveCleanedStyles = async (editor, results) => {
+  const result = results._postcssResult;
+  const styles = result.root.toString(result.opts.syntax);
+
+  if (styles === editor.getText()) {
+    // break to prevent infinite loop
```

Comment:
Maybe it would be better if fix attempts are enabled to save the state over in `lint()` and when it's called again if the current text is the same as the last end text it just immediately returns. That way we only have a few ms of overhead for this case, instead of the ~40 ms it takes to determine configuration and run the `stylelint` job and _then_ compare that the current code does.